Add note about free-tier DS limit (#118)
seqeralabs/docs: platform_versioned_docs/version-24.1.1/data/data-studios.mdx

Data Studios is a unified platform where you can host a combination of container images and compute environments for interactive tertiary analysis using your preferred tools, like JupyterLab and RStudio Notebooks, or Visual Studio Code IDEs. Each data studio session is an individual interactive environment that encapsulates the live environment for dynamic data analysis.

> On Seqera Platform Cloud, the free-tier allows only a single running data studio session at a time. To run simultaneous data studios, [contact Seqera][contact] for more information about Seqera Cloud Pro.

Reviewer: ```suggestion
On Seqera Platform Cloud, the free tier permits only one running data studio session at a time. To run simultaneous data studios, [contact Seqera][contact] for a Seqera Cloud license.
```

Reviewer 2: I would simplify to just:

```
On Seqera Cloud, the free tier permits one running data studio session at a time. To run simultaneous data studios, [contact Seqera][contact] for a Seqera Cloud Pro license.
```
